claat/parser/gdoc/parse.go

```diff
@@ -701,9 +705,10 @@ func image(ds *docState) types.Node {
 	if errorAlt != "" {
 		n.Alt = errorAlt
 	} else {
-		n.Alt = nodeAttr(ds.cur, "alt")
+		n.Alt = alt
 	}
-	n.Title = nodeAttr(ds.cur, "title")
+	// Author-added double quotes in attributes break html syntax
+	n.Title = strings.Replace(nodeAttr(ds.cur, "title"), `"`, "&qout;", -1)
```

Comment:
Should this be quot?

Reply:
(another instance of this on L675)

Reply:
oh this is embarrassing... I'll fix that